### ISSUE TITLE
false positive, interface implemented multiple times

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2015]
Quite bad OO, but anyhow, its a false positive

```abap
INTERFACE top.
  DATA bar.
ENDINTERFACE.

INTERFACE sub.
  INTERFACES top.
ENDINTERFACE.

CLASS bar DEFINITION.
  PUBLIC SECTION.
    INTERFACES top.
    INTERFACES sub.
ENDCLASS.
CLASS bar IMPLEMENTATION.
ENDCLASS.
```
